### PR TITLE
rework the rest of the pages

### PR DESCRIPTION
**Pull request by ryanlerch**
Here is the rest of the pages for elections restyled.

I have tried to reduce the number of pages a little.

Tests are going to be very broken, i want to get opinions on the new pages layout before going through and reworking all the tests.

### CODE REVIEW

**Comment by pypingou** [comment]
One issue for me: 

- Where is the link to edit an election?

**Comment by pypingou** [comment]
Could we see at limiting the size of the footer?

**Comment by pypingou** [comment]
The graph showing the evolution of the votes in the result page seems to have disappeared

**Comment by pypingou** [comment]
The cut-off mark seems to be gone from the result page as well (cf the red bar in https://admin.fedoraproject.org/voting/results/fesco-july-2016)

**Comment by pypingou** [comment]
The front-page looks really a little empty when there are no open or upcoming elections.

I think that's all I have for now :)
It's looking good though, looking forward making a release with this :)

**Comment by ryanlerch** [comment]
@pypingou the links to the elections will be on the main page as an edit link next to each election. It is also at the top of each election edit page.

I changed it so you now edit the elections on the page that you also view the election as admin. rather than having a seperate admin view / edit page, you juse submit the form from there.

**Comment by ryanlerch** [comment]
Made the graph on the results page work again, also reworked the results page to look neater, and added back the red cut-off line.

Not 100% sure what you mean by the footer, do you just want to make the min-height of the page a little bigger? the footer is itself pretty small, just on small pages the background colour of the page being set to the same colour as the footer makes it seem like the footer is large.

Also, with the front page being empty, we could just get rid of the tabs altogether, and just put the list of all the completed elections on the front page too, maybe?

other than that, i think this is right to go!

**Comment by pypingou** [comment]
Bugs:
- when editing an election, the jquery calendar widget doesn't work (when you try to change the start/end date)
- the drop-down where the values of the vote are on the vote page seems to be over the boxes of the lines, do you see what I mean? Otherwise I can provide a screenshot
- select voting and irc voting (+1/0/-1) look broken
- tests are partly broken as well :)

Questions:
- On the edit page, did you try a two column approach? Or maybe hide the 'Election details' section by default and let it unfold when clicked on: accordion ?
- I wonder if we should link to some doc describing the different election types, seems a little "dry" otherwise I think
- Do we want to keep the "heatmap" on the "confirm your vote" page? In a way I liked it since it was a little more visual but I'm not going to fight for it :)

For the footer, it's like what we did for pagure, if you see a page where the content is not that big, the footer gets big.

**Comment by ryanlerch** [comment]
@pypingou 

I have fixed the date picker on the edit election page.

the select voting and IRC voting is now fixed too.

the edit page is now a two-colomn layout as well

i have also begun fixing the tests, but hit a snag going through. It seems the admin groups aren't showing up in the edit form after saving them. I tried to trouble shoot this, but couldnt figure out what was going on.

I did have a play with the heatmaps, but decided to not include them, primarily becuase the colours were a little hard to parse conceptually -- does red mean "hot" as in my top pick, or does red mean "warning", i.e. a lower score.

**Comment by ryanlerch** [comment]
Okies, figured out the snag. It was an error i introduced.

Got all the tests working now, and this one should be right for a re-review @pypingou

**Comment by ryanlerch** [comment]
Okies, figured out the snag. It was an error i introduced.

Got all the tests working now, and this one should be right for a re-review @pypingou

File: fedora_elections/admin.py [hunk 1]
```diff
@@ -126,25 +126,13 @@ def admin_new_election():
...
-
-
-@APP.route('/admin/<election_alias>/edit', methods=('GET', 'POST'))
-@election_admin_required
-def admin_edit_election(election_alias):
-    election = models.Election.get(SESSION, alias=election_alias)
-    if not election:
-        flask.abort(404)
-
+    print election.admin_groups_list
```
Comment (pypingou) [inline fedora_elections/admin.py]: I think this one can be dropped :)

File: fedora_elections/__init__.py [hunk 2]
```diff
@@ -233,17 +243,41 @@ def index():
...
 def about_election(election_alias):
     election = models.Election.get(SESSION, alias=election_alias)
-
+    stats=[]
+    evolution_label = []
+    evolution_data = []
     if not election:
         flask.flash('The election, %s,  does not exist.' % election_alias)
         return safe_redirect_back()
+    elif election.status == 'Embargoed' or election.status == 'Ended':
```
Comment (pypingou) [inline fedora_elections/__init__.py]: maybe just ``elif election.status in ['Embargoed', 'Ended']``?

**Comment by ryanlerch** [comment]
@pypingou thanks again!

updated the PR changing those two cleanups

File: fedora_elections/admin.py [hunk 3]
```diff
@@ -213,10 +206,12 @@ def admin_edit_election(election_alias):
...
     form.lgl_voters.data = ', '.join(election.legal_voters_list)
+
     return flask.render_template(
-        'admin/election_form.html',
-        form=form,
-        submit_text='Edit election')
+            'admin/view_election.html',
+            election=election,
+            form=form,
+            submit_text='Edit election')
```
Comment (pypingou) [inline fedora_elections/admin.py]: Over-indented?

**Comment by pypingou** [comment]
Alright, thanks for fixing this so quickly :)

Let's merge and we'll fix the rest as we see them, 👍 for me!

**Comment by ryanlerch** [comment]
thanks @pypingou!

merging now!
